Microverse Hello Project

Newtayo opened the pull request:
This is a simple html and css project with linters

Newtayo committed:
updating the development-features

huseyinbiyik reviewed (changes requested):
Hi @Newtayo,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
![Almost there](https://media.giphy.com/media/Rv6GGaNrRbtNConlYo/giphy.gif)

## Highlights
- Linter checks pass 🟢
- Github flow followed 👌
- .gitignore used to ignore required files 📁

## Required Changes ♻️
- Kindly use [Microverse Readme template](https://github.com/microverseinc/readme-template) and make sure all the main sections present in yours and placeholders are replaced with your project and personal data. Only optional section is FAQ. Please remove the template logo on the top, if you have your own logo you can replace it with this one, otherwise please delete it. This is crucial to have a professional and instructive repository. 👌

-  Please make sure your commit messages are meaningful and descriptive for your next commits. Use imperative tense and capitalize the first word. For example, 
👌"Add logo to navigation bar"
👌"Fix close button functionality on mobile navbar"
❌ "Added logo to navigation bar"
❌ "Added HTML"

- Kindly fix your Pull Request title as requested [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md), it should have a descriptive title telling you what changes are inside. Also please fix the Pull Request description section, it should have a short summary describing the introduced changes. 
<img width="885" alt="Screenshot 2023-01-10 at 11 19 19 PM" src="https://user-images.githubusercontent.com/25887253/211653696-ac2c8c0f-6edb-47a6-b72e-1a044c765eb3.png">

- Please use descriptive branch names instead of "development-features" (e.g.: refactor-authentication, user-content-cache-key, make-retina-avatars), so that others can see what is being worked on. [Check](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md) more to learn more about this. For example, "setup-project" is a more specific and more descriptive one.


_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Newtayo commented:
Microverse Hello Project Pull Request 1

In this project, the following was done:

Created a HTML and Style sheet files.
Incorporated Microverse HTML and CSS linters for code validation
-Added gitignore file to prevent the node/modules and test.md files from being commit to git repository
